ui/raidboss/common_replacement.js

```diff
@@ -45,7 +45,7 @@ let commonReplacement = {
   },
   'Engage!': {
     de: 'Start!',
-    fr: 'À l\'attaque',
+    fr: 'À l\'attaque !',
```

Comment:
Missing exclamation mark, with a space in French

Reply:
Sorry, but please don't include the exclamation point here.  See #57 for more details.

Reply:
Wow, this was not easy. 
- Was this option not retained?
![engage](https://user-images.githubusercontent.com/61565834/79203820-de8a5780-7e3b-11ea-8ec5-d2d9133cc50d.PNG)

Reply:
Y'en a pas besoin car ça parse "À l'attaque" peu importe ce qu'il y a après. On a fait pas mal de test il y a un moment et c'est comme ça que ça fonctionne le mieux :)

Reply:
the correction was made as requested. Why don't you answer me on the discord? 
Thank you for your answers